Darkly/EventSource/EventSource.h

```diff
@@ -0,0 +1,100 @@
+//
```

Comment:
Why is this inlined rather than specified as a CocoaPod dependency?

https://cocoapods.org/pods/EventSource